README.md

```diff
@@ -580,6 +580,7 @@ See https://github.com/n1trux/awesome-sysadmin#distributed-filesystems
 
 #### File transfer/synchronization
 
+  * [Files Sharing](https://github.com/axeloz/filesharing) - Open Source and self-hosted files sharing application based on unique and temporary links..  `GPL-3.0` `PHP`
```

Comment:
* Please move it to https://github.com/Kickball/awesome-selfhosted#single-clickdrag-n-drop-upload
 * License: `GPLv3`
 * Please remove the double dot and double space at the end of the description